db/models.py

```diff
@@ -22,6 +27,11 @@ class Movie(models.Model):
     actors = models.ManyToManyField(to=Actor)
     genres = models.ManyToManyField(to=Genre)
```

Comment:
Specify `related_name` in both fields

Reply:
You can find an example in the [checklist](https://github.com/mate-academy/py-taxi-service-startproject/blob/master/checklist.md)